examples/jsm/objects/WaterMesh.js

```diff
@@ -51,7 +51,7 @@ class WaterMesh extends Mesh {
 		 * @type {number}
 		 * @default 0.5
 		 */
-		this.resolution = options.resolution !== undefined ? options.resolution : 0.5;
+		this.resolutionScale = options.resolutionScale !== undefined ? options.resolutionScale : undefined;
```

Comment:
The effect is currently broken because `undefined` isn't a valid value. It should be
```js
this.resolutionScale = options.resolutionScale !== undefined ? options.resolutionScale : 0.5;
```

For testing: https://rawcdn.githack.com/jonsmithers/three.js/smithers/fix-field-assignment/examples/webgpu_ocean.html

Reply:
Oh neat! It should be fixed but I'm not sure why that url isn't updating 🤔 

(I _did_ test this locally (localhost:8080/examples/webgpu_ocean.html), but I was testing with various hard-coded values, not the default value 😬)